routes/routes.js

```diff
@@ -77,11 +85,6 @@ router.post("/user/address", postUserAddress);
 //userListAddress
 router.get("/user/get/:id", validateJwtToken, getUserListAddressById);
 //uploadImage
-router.post(
-  "/user/profile",
-  upload.single("profile"),
-  validateUserProfile,
-  userProfile
-);
+router.post("/user/profile", upload.single("profile"), userProfile);
```

Comment:
can you add JWT authentication in route "/user/profile" @mukeshsin

Reply:
ok i do